README.md

```diff
@@ -161,10 +211,11 @@ Ref issue [#21](https://github.com/jarun/googler/issues/21).
 1. Copyright (C) 2008 Henri Hakkinen
 2. Resurrected and maintained (2015 -) by [Arun Prakash Jana](mailto:engineerarun@gmail.com)
 3. [Zhiming Wang](https://github.com/zmwangx)
-  
+
 Special thanks to [jeremija](https://github.com/jeremija), [shaggytwodope](https://github.com/shaggytwodope) and [Narrat](https://github.com/Narrat) for their contributions and efforts in spreading `googler`.
 
-<h1 id="Notes">Notes</h1>
+# Notes
 
 1. Initially I raised a pull request but I could see that the last change was made 7 years earlier. In addition, there is no GitHub activity from the original author [Henri Hakkinen](https://github.com/henux) in past year. I have created this independent repo for the project with the name `googler`. Would love to push the changes back to original repo if the author contacts. I retained the original copyright information.
```

Comment:
Let's drop the line: Would love to push the changes back to original repo if the author contacts.
I won't enjoy it much at this stage. Anyone can fork and modify on their own.